packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -1784,8 +1784,8 @@ module.exports = {
    * @returns {Boolean}
    */
   isPluginDocumentationNeeded: function(pluginName) {
-    const pluginsDocWantedByUser = strapi.plugins.documentation.config['x-strapi-config'].pluginsForWhichToGenerateDoc;
-    if (Array.isArray(pluginsDocWantedByUser) && !pluginsDocWantedByUser.includes(pluginName)) {
+    const pluginsForWhichToGenerateDoc = strapi.plugins.documentation.config['x-strapi-config'].pluginsForWhichToGenerateDoc;
```

Comment:
You could have written i tlike so:

`const { pluginsForWhichToGenerateDoc } = strapi.plugins.documentation.config['x-strapi-config'];`

;)

Reply:
let me do that